src/modules/shared/components/MediaCard/MediaCard.tsx

```diff
@@ -165,10 +168,30 @@ const MediaCard: FC<MediaCardProps> = ({
 		);
 	};
 
+	const renderKeyUserPill = () => {
```

Comment:
Add a test to check if this block is rendered correctly eg `Should render the key user pill when...`